scripts/src/modules/analytics/article_tracking/remove_mobile_tracking.js

```diff
@@ -11,7 +11,9 @@ export default function remove_mobile_tracking() {
         item.removeAttribute("data-link");
     });
 
+    /* eslint-disable no-undef */
     article_body.removeEventListener("click", (e) => {
         mobile_section_tracking(e);
     });
+    /* eslint-enable no-undef */
```

Comment:
I cannot find this function declared anywhere, but am wary of just removing this code

Reply:
Again, this might all be deleted or re-written eventually. Leave it in for now to save the headaches!